src/arrayOfPeople.js

```diff
@@ -38,7 +38,11 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  const peopleArr = [];
+  people.forEach(person => peopleArr.includes(person) ? 
```

Comment:
How to format arrow function with ternary:
```javascript
  people.forEach(person => (
    peopleArr.includes(person)
      ? peopleArr.splice(peopleArr.indexOf(person), 1)
      : peopleArr.push(person)
  ));
```

Or without additional parenthesis but for my mind 1st one is better:


```javascript
  people.forEach(person => peopleArr.includes(person)
    ? peopleArr.splice(peopleArr.indexOf(person), 1)
    : peopleArr.push(person)
  );
```